plugins/USBPrinting/USBPrinterOutputDevice.py

```diff
@@ -69,6 +69,11 @@ def __init__(self, serial_port, baud_rate = None):
     #   is ignored.
     #   \param kwargs Keyword arguments.
     def requestWrite(self, nodes, file_name = None, filter_by_machine = False, file_handler = None, **kwargs):
+        if self._is_printing:
+            return # Aleady printing
+
+        Application.getInstance().showPrintMonitor.emit(True)
```

Comment:
Once the plugins are updated we can remove this emit right? It's only there for backwards compatibility (as we have the setStage() method now).

Reply:
Yeah, i'll change it now.

Reply:
see bd4797404df6761c84bf1b8b7f160957f8199805